src/iotjs.c

```diff
@@ -122,6 +122,23 @@ bool iotjs_initialize(iotjs_environment_t* env) {
   return true;
 }
 
+void iotjs_restart(iotjs_environment_t* env, jerry_value_t jmain) {
+  jerry_value_t abort_value = jerry_get_value_from_error(jmain, false);
+  if (jerry_value_is_string(abort_value)) {
+    static const char restart_str[] = "r353t";
```

Comment:
Is this a jerry debugger feature, or else a feature only for vscode iotjs extension? It seems better to directly refer to the value defined in any of them if possible.

Reply:
We could add an api function to check this. Sounds like a good idea.

Reply:
As adding an extra jerry api function would require also a submodule update here. I would put a TODO mark about it here and merge this PR as soon as possible. In the mean time the api function can be added.